ingest/pom.xml

```diff
@@ -203,6 +193,18 @@
             <version>${sqlite-jdbc.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
```

Comment:
Is this "api" package really necessary if the full package is already included (immediately above)?

Reply:
We do need it, otherwise it complains about "implicit dependency"

Reply:
I vaguely remember already fixing this. In master, we don't have both dependencies - we have just the api.

Reply:
Ok I removed it here to.
The order is different because here it's alphabetical